unime/src/routes/(app)/scan/+page@.svelte

```diff
@@ -187,7 +187,7 @@
     console.log('onMount: /scan');
     document.documentElement.querySelector('body')!!.classList.add('transparent');
     // permissionsGiven = await checkScanPrerequisites();
-    if (!$state?.dev_mode_enabled) {
+    if ($state?.dev_mode !== 'Off') {
```

Comment:
This is a major logical bug. `startScan()` should be called when `dev_mode` is not enabled. You flipped the logic to "when `dev_mode` is not Off" (meaning the scanner now starts directly when `dev_mode` is enabled and does not work anymore for regular users). Didn't you notice when you tested it on your device? I assumed so since you checked the box in the Definiton of Done: _I have performed a self-review of my own code_

Reply:
Well self-review I might interpret differently, but I will build on my phone and test. However that same logical bug was there before only it would break on dev mode not other way around. So we definitely need a clean solution for this which I will think of.